mimic_2_data_analysis.ipynb

```diff
@@ -0,0 +1,3433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    newdat =  expand_grid(dictionary)
</pre><p><code>newdat</code> is also not the best of all names (see comment above). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>